pkg/registry/flowcontrol/rest/storage_flowcontrol.go

```diff
@@ -101,70 +100,80 @@ func (p RESTStorageProvider) GroupName() string {
 
 // PostStartHook returns the hook func that launches the config provider
 func (p RESTStorageProvider) PostStartHook() (string, genericapiserver.PostStartHookFunc, error) {
-	return PostStartHookName, func(hookContext genericapiserver.PostStartHookContext) error {
-		flowcontrolClientSet := flowcontrolclient.NewForConfigOrDie(hookContext.LoopbackClientConfig)
+	return PostStartHookName, ensureAPFBootstrapConfiguration(), nil
+
+}
+
+func ensureAPFBootstrapConfiguration() genericapiserver.PostStartHookFunc {
```

Comment:
just do:

func ensureAPFBootstrapConfiguration(hooContext ..) error {
}

and in line 103 instead of calling this function just return it.